+ source ../common/storage_disk_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environment preparation.'
+ message='Start environment preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environment preparation.'
Fri Apr 28 07:14:31 2023 - INFO  - Start environment preparation.
+ check_free_disk
++ TEST_DISK
++ id=1
++ python3 /root/mugen/libs/locallibs/get_test_device.py --device disk --node 1
+ disks=vdb
+ disk_list=($disks)
+ local_disk=vdb
+ local_disk1=
+ local_disk2=
+ local_disk3=
+ fdisk /dev/vdb
+ echo -e 'n\np\n1\n\n+1G\np\nw\n'

Welcome to fdisk (util-linux 2.37.2).
Changes will remain in memory only, until you decide to write them.
Be careful before using the write command.

Device does not contain a recognized partition table.
Created a new DOS disklabel with disk identifier 0x4d90e1ea.

Command (m for help): Partition type
   p   primary (0 primary, 0 extended, 4 free)
   e   extended (container for logical partitions)
Select (default p): Partition number (1-4, default 1): First sector (2048-1023999, default 2048): Last sector, +/-sectors or +/-size{K,M,G,T,P} (2048-1023999, default 1023999): Value out of range.
Last sector, +/-sectors or +/-size{K,M,G,T,P} (2048-1023999, default 1023999): Last sector, +/-sectors or +/-size{K,M,G,T,P} (2048-1023999, default 1023999): Last sector, +/-sectors or +/-size{K,M,G,T,P} (2048-1023999, default 1023999): 
Created a new partition 1 of type 'Linux' and of size 499 MiB.

Command (m for help): 
+ mkfs.ext4 -F /dev/vdb1
mke2fs 1.46.5 (30-Dec-2021)
The file /dev/vdb1 does not exist and no size was specified.
+ mount /dev/vdb1 /mnt
mount: /mnt: special device /dev/vdb1 does not exist.
+ LOG_INFO 'Environmental preparation is over.'
+ message='Environmental preparation is over.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Environmental preparation is over.'
Fri Apr 28 07:14:41 2023 - INFO  - Environmental preparation is over.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start executing testcase!'
+ message='Start executing testcase!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start executing testcase!'
Fri Apr 28 07:14:42 2023 - INFO  - Start executing testcase!
+ umount /dev/vdb1
umount: /dev/vdb1: no mount point specified.
+ CHECK_RESULT 32
+ actual_result=32
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 32 ']'
+ '[' 0 -eq 0 ']'
+ test 32x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_fileCMD_dd.sh line 32'
+ message='oe_test_storage_fileCMD_dd.sh line 32'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_fileCMD_dd.sh line 32'
Fri Apr 28 07:14:43 2023 - ERROR - oe_test_storage_fileCMD_dd.sh line 32
+ '[' 0 -eq 1 ']'
+ return 0
+ echo y
+ sudo mkfs -t ext3 /dev/vdb1
mke2fs 1.46.5 (30-Dec-2021)
The file /dev/vdb1 does not exist and no size was specified.
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_storage_fileCMD_dd.sh line 34'
+ message='oe_test_storage_fileCMD_dd.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_storage_fileCMD_dd.sh line 34'
Fri Apr 28 07:14:44 2023 - ERROR - oe_test_storage_fileCMD_dd.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ dd if=/dev/zero of=/dev/vdb1 count=1 bs=512
1+0 records in
1+0 records out
512 bytes copied, 0.00698278 s, 73.3 kB/s
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ dd --help
+ grep -i Usage
Usage: dd [OPERAND]...
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End of testcase execution!'
+ message='End of testcase execution!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of testcase execution!'
Fri Apr 28 07:14:45 2023 - INFO  - End of testcase execution!
+ CASE_RESULT 0
+ case_re=0
+ test -z 2
+ test 2 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Apr 28 07:14:46 2023 - ERROR - failed to execute the case.
+ exit 2
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri Apr 28 07:14:46 2023 - INFO  - start environment cleanup.
+ echo -e 'd\np\nw\n'
+ fdisk /dev/vdb

Welcome to fdisk (util-linux 2.37.2).
Changes will remain in memory only, until you decide to write them.
Be careful before using the write command.

Device does not contain a recognized partition table.
Created a new DOS disklabel with disk identifier 0xdfa32fe7.

Command (m for help): No partition is defined yet!

Command (m for help): 
Disk /dev/vdb: 500 MiB, 524288000 bytes, 1024000 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0xdfa32fe7

Command (m for help): The partition table has been altered.
Calling ioctl() to re-read partition table.
Syncing disks.

+ LOG_INFO 'Finish environment cleanup.'
+ message='Finish environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup.'
Fri Apr 28 07:14:47 2023 - INFO  - Finish environment cleanup.
